release-notes/unreleased/IGAPP-83-ShowFooterOnceLoaded.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-83
+show_in_stores: false
+platforms:
+  - web
+en: Footer and Header are only shown once everything is ready to prevent penalties by google.
```

Comment:
to prevent penalties from google. Does that need to be part of the commit message? Sounds like we are criminals 🙈

Reply:
Do you have a better suggestion? :D

Reply:
You could just remove that part. Release notes should not have to proclaim why they are there, don't they?